### DISCUSSION TITLE
请问next表里的task只能从头到尾遍历吗

### DISCUSSION THREAD

**Discussion by Guili-Ad** [MaaXYZ/MaaFramework#234] [Q&A]
向大佬们请教一下，如果我一个task的next列表里有[1, 2, 3 ]三个task，在json低代码模式里有办法让1以及后续任务执行之后，返回到这个[1, 2, 3]的列表的时候，跳过1不去识别，直接从2开始吗？还是说json低代码只能做线性流程呀orz
主要是因为1，2，3这三个任务链只有最开始的识别不同，后续动作都是完全一样的，写纯线性好像会多一些冗余代码（orz

**Comment by MistEO** [comment]
返回 [1,2,3] 可以用 is_sub 字段（但做不到跳过识别 1）

或者可以考虑做成两个入口：先执行 A: [1,2,3]，执行完再执行 B: [2,3]。

再或者就是自己写控制流了 [开发思路](https://github.com/MaaXYZ/MaaFramework/blob/main/docs/zh_cn/1.1-%E5%BF%AB%E9%80%9F%E5%BC%80%E5%A7%8B.md#%E5%BC%80%E5%8F%91%E6%80%9D%E8%B7%AF)
